packages/e2e-tests/specs/site-editor/multi-entity-saving.test.js

```diff
@@ -173,19 +160,18 @@ describe( 'Multi-entity save flow', () => {
 			await page.keyboard.type( '...more title!' );
 
 			// Verify update button is enabled.
+			await page.waitForSelector( enabledSavePostSelector );
 			const enabledSaveButton = await page.$( enabledSavePostSelector );
```

Comment:
There seems to be a problem here, although it was always passing locally 😄 . I'll need to investigate more